src/ILLink.RoslynAnalyzer/Resources.resx

```diff
@@ -121,7 +121,7 @@
     <value>RequiresUnreferencedCodeAnalyzer</value>
   </data>
   <data name="RequiresUnreferencedCodeAnalyzerMessage" xml:space="preserve">
-    <value>Calling '{0}' which has `RequiresUnreferencedCodeAttribute` can break functionality when trimming application code. {1}. {2}</value>
+    <value>'{0}' method has 'RequiresUnreferencedCodeAttribute' which can break functionality when trimming application code. {1}. {2}</value>
```

Comment:
Maybe we could just say "Using method which has RequiresUnreferencedCode ..." - it reads better to me (and it's a bit more precise - the linker really thinks the app is going to use the method"

Reply:
I'll change this -- I was originally following the shape that we already have for warnings related to Dynamically Accessed Members: https://github.com/mono/linker/pull/1624.